src/http/request.rs

```diff
@@ -167,13 +194,9 @@ impl Request {
 
     /// Get Module context
     pub fn get_module_ctx<T>(&self, module: &ngx_module_t) -> Option<&T> {
-        let cf = self.get_module_ctx_ptr(module) as *mut T;
-
-        if cf.is_null() {
-            return None;
-        }
-        let co = unsafe { &*cf };
-        Some(co)
+        let ctx = self.get_module_ctx_ptr(module) as *const T;
```

Comment:
If we are using `.cast::<T>()` to cast to `*const T` above, shouldn't we use the same here?

Reply:
`cast()` actually converts to `*mut T` and makes the debug code one (`mov`) instruction longer.
The release code looks the same though, so I'll make this change.